.github/styles/config/vocabularies/Docs/accept.txt

```diff
@@ -1,89 +1,92 @@
-(?i)API
-(?i)app
-(?i)AutoTrain
-(?i)aws
-(?i)autogenerate
-(?i)chatbot
-(?i)automation
-(?i)autolog
-(?i)basename
-(?i)bokeh
-(?i)boolean
-(?i)checkpoint
+API
+[Aa]pps?
+AutoTrain
+AWS
```

Comment:
AWS sort of ruins this nice syntax. E.g. an AWS [ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) will posses lower case `aws`. Maybe keep as is?